src/include/mpi.h.in

```diff
@@ -421,6 +421,7 @@ typedef MPI_Session_errhandler_function MPI_Session_errhandler_fn;
    in 22-25), index in the low bits */
 #define MPI_ERRORS_ARE_FATAL ((MPI_Errhandler)0x54000000)
 #define MPI_ERRORS_RETURN    ((MPI_Errhandler)0x54000001)
+#define MPI_ERRORS_ABORT     ((MPI_Errhandler)0x54000003)
```

Comment:
Is there any other implementation needed to support this? What happens if a user set the errhandler to `MPI_ERRORS_ABORT` on an object?

Reply:
Right, we probably should add code to treat `MPI_ERRORS_ABORT` the same as `MPI_ERRORS_ARE_FATAL`, at least for now.

Reply:
done.

Reply:
@hzhou Perhaps you should open a new issue to remember that proper semantics for this new error handler are yet to be implemented?

Reply:
I have made `MPI_ERRORS_ABORT` work the same way as `MPI_ERRORS_ARE_FATAL` now. For most cases that probably is what user wanted. There are a few remaining items to make session truely isolated from each other -- that may be tricky. Opened https://github.com/pmodels/mpich/issues/5427 for remaining sessions related items.